src/App.js

```diff
@@ -264,157 +265,158 @@ class App extends React.Component {
-                    <InstantSearch
-                      appId={ALGOLIA_APP_ID}
-                      apiKey={ALGOLIA_SEARCH_KEY}
-                      indexName="roles"
-                    >
-                      <Index indexName="names">
-                        <Configure />
-                        <CurrentRefinements
-                          transformItems={items => items.filter(item => item.public === 'true' || item.public === 'false')}
-                        />
-                        <VirtualMenu attributeName="public" defaultRefinement={'true'} />
-                      </Index>
-                      <Index indexName="vendors">
-                        <Configure />
-                        <CurrentRefinements
-                          transformItems={items => items.filter(item => item.public === 'true' || item.public === 'false')}
-                        />
-                        <VirtualMenu attributeName="public" defaultRefinement={'true'} />
-                      </Index>
-                      <Index indexName="locations">
-                        <Configure />
-                        <CurrentRefinements
-                          transformItems={items => items.filter(item => item.public === 'true' || item.public === 'false')}
-                        />
-                        <VirtualMenu attributeName="public" defaultRefinement={'true'} />
-                      </Index>
-                      <Index indexName="locationTypes" />
-                      <AutoCompleteBar
-                        onUpdateInput={query => this.searchQuery = query}
-                        onEnterHit={() => {
-                          if (this.searchQuery) {
-                            history.push({ pathname: '/search', search: `?query=${encodeURIComponent(this.searchQuery)}&show=all` })
-                          }
+        <Grid fluid style={{ padding: 0 }}>
+          <AppBar
+            style={{ ...appBarStyle }}
+            iconElementLeft={
+              <div>
+                <Row>
+                  <Col xs>
+                    <Link to="/"><img src={Logo} className="logo" alt="Film Indy Logo" /></Link>
+                  </Col>
+                  { location.pathname !== '/' &&
+                    <Col xs>
+                      <Card className="menuSearchCard">
+                        <div style={{ display: 'flex', flexDirection: 'row' }}>
+                          <SearchIcon className="searchIcon" />
+                          <InstantSearch
+                            appId={ALGOLIA_APP_ID}
+                            apiKey={ALGOLIA_SEARCH_KEY}
+                            indexName="roles"
+                          >
+                            <Index indexName="names" />
+                            <Index indexName="vendors" />
+                            <Index indexName="locations" />
+                            <Index indexName="locationTypes" />
```

Comment:
@ejmudrak there is a major difference here. This is where the virtual menu is being used. In this PR, it is being removed. It shouldn't be.